transformations/negate_strengthen/test.json

```diff
@@ -0,0 +1,122 @@
+{
+  "type": "NegateStrengthen",
+  "test_cases": [
+    {
+      "class": "NegateStrengthen",
+      "inputs": {
+        "sentence": {
+          "sentence": "1200 Japanese Canadians were sent to Greenwood because of the Japanese Canadian internment in 1942 .",
+          "signal_id": 7
```

Comment:
The first two test cases are different from the other test cases and seem to be wrongly formatted.